wrangler/src/@utils/trust.js

```diff
@@ -86,15 +86,24 @@ export function checkLocalQuorum(turret) {
  * @param {string} oldTurret - the address of the turret that is being removed from the source account
  * @param {string} newTurret - the new turret to add to the source account
  * @param {string} functionHash - the hash of the function that is being healed
- * @returns {Promise<string>} The transaction XDR, the signer public key, and the signature.
+ * @param {number} timestamp - The heal event timestamp
+ * @param {string} userPublicKey - The publicKey / accountId of the user running heal
+ * @param {string} fee - The fee, in stroops, to use for the transaction
+ * @returns {Promise<any>} The transaction XDR, the signer public key, and the signature.
  */
-export async function heal(controlAccount, oldTurret, newTurret, functionHash) {
+export async function heal(
+  controlAccount,
+  oldTurret,
+  newTurret,
+  functionHash,
+  timestamp,
+  userPublicKey,
+  fee
+) {
```

Comment:
did not update these names to make the dif easier to track.

I'm indifferent about the names being consistent with the API layer, but am happy to change in a separate commit.

Reply:
i'm fine with the names being updated.  I feel like timestamp should be timeout though.